packages/documentation/src/pages/platform/architecture/cms.mdx

```diff
@@ -45,6 +45,40 @@ Drupal is queried in the Jenkins build process for any vets-website branch. On J
 - [Jenkins build process flow](/platform/architecture/build-deploy-flows#standard-masterpr-build)
 - [Drupal/AWS cache script](https://github.com/department-of-veterans-affairs/vets-website/tree/master/script/drupal-aws-cache.js)
 
+### Automated accessibility checking
+
+All pages created and managed with the content management system are required to pass accessiblity scans for [WCAG2 Level A and AA](https://www.w3.org/TR/WCAG20/) compliance. The Jenkins build process automatically scans every page, and will break on Section 508 or WCAG violations. To help prevent this, we have [created an accessibility check](https://github.com/department-of-veterans-affairs/vets-website/pull/10836) that can be used on `localhost` and the CMS preview servers.
+
+#### Reviewing issues on localhost
+
+1. Start your SOCKS proxy
+2. Open a terminal window and type the following command except the dollar sign:
+3. `$ npm run build -- --pull-drupal --entry=static-pages --accessibility`
```

Comment:
>[Fence code blocks with three back ticks and language for syntax highlighting](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/documentation-guide/documentation-style-guide)

so this should be something like: 

2. Open a terminal window
3. Build Va.gov locally (run this command in the terminal): 
```
npm run build -- --pull-drupal --entry=static-pages --accessibility
```